### ISSUE TITLE
need new fanart

### ISSUE THREAD

**Issue by primaeval** [vonH/plugin.video.iplayerwww#184]
We could do with a new fanart image. Several skins and websites use the fanart as a widescreen logo. See the Kodi Titan skin for instance. The blurry one doesn't really look too good. 
https://kodi.tv/addon/music-add-ons-plugins-video-add-ons/iplayer-www

**Comment by Mike-KL** [comment]
My vote is for use of standard iplayer logo similar to

![iplayer](https://cloud.githubusercontent.com/assets/16574667/26527299/33a72e34-4391-11e7-8f03-87056b387322.jpg)

**Comment by CaptainTK** [comment]
I am happy to hear suggestions. However, they must not be prone to copyright issues, which disqualifies your idea, right away @Mike-KL .

BTW: I have no idea how the image got in the add-on gallery.

**Comment by primaeval** [comment]
What is the icon.png copyright situation? 
If that is ok then Mike-KL's idea is just to move the names around in Photoshop or Gimp.

**Comment by primaeval** [comment]
Here is one I made based on the wikipedia svg file.
https://en.wikipedia.org/wiki/BBC_iPlayer#/media/File:BBC_iPlayer_logo.svg
![iplayer](https://cloud.githubusercontent.com/assets/11716348/26623969/b4d1d0bc-45ef-11e7-9c58-8ca807f71a9d.jpg)

License
https://en.wikipedia.org/wiki/Wikipedia:Text_of_Creative_Commons_Attribution-ShareAlike_3.0_Unported_License

**Comment by Mike-KL** [comment]
@CaptainTK So if your not comfortable with "BBC Player" directly why not "iplayer WWW" using same text font color and background format etc?

@primaeval @CaptainTK, already downloaded above and replaced my existing fan-art in order to see to see how it would look in kodi as a background

**Comment by CaptainTK** [comment]
Interesting idea, @primaeval. I will need to look into this in a little more detail, but if the license at Wikipedia checks out, I am happy to use this in the future.

**Comment by primaeval** [comment]
The Titan skin and variants are the real problem in my opinion. I don't know how many skins use the fanart as the main icon but Titan is quite popular. 
I had a lot of blank plain black fanarts for some of my addons before I realised what Titan was using.

**Comment by CaptainTK** [comment]
OK, I checked the license of said file on Wikipedia. Unfortunately, it states "BBC" as author and contains a copyright warning:

> This work of art is copyrighted in its source country until copyright expiry conditions have been met. Information on this image (Creator, death date, etc) should always be listed if known. This image should not be transferred to Wikimedia Commons unless it can be verified to be in the public domain in its country of first publication.

I don't know, where you found the information this was under CC license @primaeval .

**Comment by primaeval** [comment]
Where are you looking?
https://en.wikipedia.org/wiki/File:BBC_iPlayer_logo.svg

**Comment by primaeval** [comment]
It clearly says Public Domain at the bottom of the page when you scroll down or look at the More Details.
https://en.wikipedia.org/wiki/BBC_iPlayer#/media/File:BBC_iPlayer_logo.svg

**Comment by CaptainTK** [comment]
I am looking at More Details and copied the quote from there.

**Comment by primaeval** [comment]
![shot-20170605-12379-1w2okxg](https://cloud.githubusercontent.com/assets/11716348/26787071/05801b64-4a09-11e7-9bcb-536ea447a678.jpeg)

**Comment by primaeval** [comment]
![shot-20170605-12379-1ty82jw](https://cloud.githubusercontent.com/assets/11716348/26787133/383cbfa8-4a09-11e7-83ea-3bbbfa596f14.jpeg)

**Comment by CaptainTK** [comment]
You got it. Now look at the warning signs on the second screenshot.

**Comment by primaeval** [comment]
1. The Kodi repo is in the USA so the Public Domain applies.
2. You'll have to remove the icon.png file as that uses this image.

**Comment by Mike-KL** [comment]
I know...its a very hasty mock up of extremely poor quailty, but could we play around with a similar colour format and name as per current addon? (perhaps dropping i from iplayer ?)

![image](https://cloud.githubusercontent.com/assets/16574667/26794762/4f674cb0-4a23-11e7-9da4-e5cbc99277ca.png)

**Comment by primaeval** [comment]
Thanks @Mike-KL but anything that doesn't look like the BBC logo appears more like the addon is stealing the BBC content and using it as it's own instead of just being a front end to the iPlayer web site in my opinion. 
If you think the icon.png is ok then my fanart.jpg must be too as they both originate from the svg file but have several layers of rasterization and bitmap manipulation from the original vector graphic.
If not then you'll have to remove the icon.png file and probably not even call the addon iPlayer as that is some kind of trademark.

**Comment by primaeval** [comment]
Here is the icon.png with the svg scaled and placed on top and offset from the original.
![icon ontop](https://cloud.githubusercontent.com/assets/11716348/26798724/fc7d260a-4a31-11e7-9fbf-98518c19ba8f.png)
![icon overlay](https://cloud.githubusercontent.com/assets/11716348/26798725/fc80256c-4a31-11e7-9086-841a64ef814a.png)

**Comment by CaptainTK** [comment]
Agreed. It looks like icon.png might no be OK after all. As I explained before, I just took it from the old add-on and trusted the licence published there.
